test/testdata/vectors/pip/vectors.go

```diff
@@ -0,0 +1,128 @@
+package pip
+
+import (
+	"github.com/nuts-foundation/nuts-knooppunt/test/testdata/vectors/hapi"
+	"github.com/zorgbijjou/golang-fhir-models/fhir-models/caramel/to"
+	"github.com/zorgbijjou/golang-fhir-models/fhir-models/fhir"
+)
+
+func HAPITenant() hapi.Tenant {
+	return hapi.Tenant{
+		Name: "policy-information-point",
+		ID:   8,
+	}
+}
+
+func CapabilityStatements() []fhir.CapabilityStatement {
```

Comment:
maybe we can do a ``//go:generate`` instruction that copies this from `component/pdp/capabilities`, and unmarshal that? Then we don't need to keep it in sync?

Reply:
Why do we even need this here?